pkgs/top-level/all-packages.nix

```diff
@@ -915,6 +915,8 @@ in
 
   burp = callPackage ../tools/backup/burp { };
 
+  buku = callPackage ../applications/misc/buku { };
```

Comment:
You can select you python version here by overriding the passed arguments like so:

```
buku = callPackage ../applications/misc/buku {
  pythonPackages = python3Packages;
};
```